Sources/MapboxSearch/PublicAPI/Search Results/ResultChildMetadata.swift

```diff
@@ -23,4 +23,27 @@ public struct ResultChildMetadata: Codable, Hashable {
         }
         self.name = resultChildMetadata.name
     }
+
+    /// Initializes a new instance of `ResultChildMetadata` with optional category, coordinates, and name,
+    /// and a required `mapboxId`.
+    ///
+    /// - Parameters:
+    ///   - category: An optional `String` representing the category of the result.
+    ///   - coordinate: An optional `CLLocationCoordinate2D` representing the geographical location
+    ///     associated with the result. If provided, it is transformed to a `CLLocationCoordinate2DCodable`
+    ///     instance for storage.
+    ///   - mapboxId: A required `String` that uniquely identifies the Mapbox object. This is a required
+    ///     parameter and must be provided during initialization.
+    ///   - name: An optional `String` representing the name of the result.
```

Comment:
The order of the parameter should be the same as it is in the method, so I guess it should be
```suggestion
    /// Initializes a new instance of `ResultChildMetadata` with optional category, coordinates, and name,
    /// and a required `mapboxId`.
    ///
    /// - Parameters:
    ///   - mapboxId: A required `String` that uniquely identifies the Mapbox object. This is a required
    ///     parameter and must be provided during initialization.
    ///   - name: An optional `String` representing the name of the result.
    ///   - category: An optional `String` representing the category of the result.
    ///   - coordinate: An optional `CLLocationCoordinate2D` representing the geographical location
    ///     associated with the result. If provided, it is transformed to a `CLLocationCoordinate2DCodable`
    ///     instance for storage.
```